projects/packen-ng/README.md

```diff
@@ -201,11 +201,11 @@ interface TableHeader {
 
 #### **ItemInfo** class description
 
- - **ItemInfo**: Defines content and has the structure `key:value`. For example `Nombre: Pedro`.
- - **a) key**: Defines the content before `:`.
- - **b) value**: Defines the content after `:`.
- - **c) showInDesktop**: Defines whether the `key` value will be displayed on the desktop or not. Possible values: `(true|false)`.
- - **d) style**: Defines box style of `value` property. Possible values: `(blue-box|gray-box|blue-gray-box)`.
+ - **ItemInfo**: It defines a content and has the structure `key:value` . For example `Nombre: Pedro` .
```

Comment:
@germanpacken prefiero que vuelves esto a como estaba ... "It defines" es un poco mal en Inglés y "Posibles" es incorrecto ....